cime_config/namelist_definition_blom.xml

```diff
@@ -7481,6 +7490,31 @@
     <desc>Shelfsea water residence time [d]</desc>
   </entry>
 
+  <entry id="lyr_tdoc_lc">
+    <type>integer(3)</type>
+    <category>diabgc</category>
+    <group>diabgc</group>
+    <values>
+      <value>0,0,2</value>
+      <value is_test="yes">0,0,2</value>
+      <value is_test="yes" empty_hist="yes">0,0,0</value>
+      <value hamocc_output_size="spinup">0,0</value>
```

Comment:
Here and below: I would at least do some yearly output also during spinup, but maybe Joerg is of different opinion? (would then be 0,2).